monitoring/uss_qualifier/scenarios/faa/uft/message_signing_start.py

```diff
@@ -51,15 +51,17 @@ def run(self):
         self.begin_test_step("Signal mock USS")
 
         # TODO: Add call to mock USS to start message signing report
+        start_query = self._mock_uss.start_msg_sign_recording()
```

Comment:
The TODO has been fulfilled so it should be removed

Reply:
Will remove.